certifier/fixtures/badreason.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Are these fixtures necessary? If you want to reuse them, cant you just use mixer.blend and specify them in conftest.py?